src/backend/app/main.py

```diff
@@ -134,6 +134,9 @@ def get_logger():
             # format=log_json_format, # JSON format func
         )
 
+        log.add("/opt/logs/create_project.log",
+                enqueue=True
```

Comment:
This needs to filter by the bound task, otherwise you will get all logs:

```python
log.add("/opt/logs/create_project.log",
    enqueue=True,
    filter=lambda record: record["extra"]["task"] == "create_project"
)
```

Reply:
Also if these logs are made to be read in by the code, and not read by people directly, I would recommend JSON logging.

```python
log.add("/opt/logs/create_project.log",
    enqueue=True,
    serialize=True,
    filter=lambda record: record["extra"]["task"] == "create_project"
)
```